assets/src/js/listing_pages.js

```diff
@@ -17,4 +18,27 @@ export const setupListingPages = () => {
 
   listViewToggle.onclick = switchViews;
   gridViewToggle.onclick = switchViews;
+
+  // Setup filters for the News & Stories page.
+  const filters = document.querySelector('.listing-page-filters');
+  if (!filters) {
+    return;
+  }
+
+  // Functions and constants.
+  const CATEGORY_FILTER = 'category';
+
+  const updateFilter = event => {
+    const {target: {id, value}} = event;
+    const urlParams = new URLSearchParams(window.location.search);
+    if (value) {
+      urlParams.set(id, value);
```

Comment:
Should we pass the category slug instead of the ID as the parameter value?

Reply:
To make it more readable?

Reply:
It would be a bit more complicated to set up the query later on, since the `category__in` parameter uses the ids

Reply:
> To make it more readable?

Yes. Also to make it more SEO friendly and to follow the [UK example](https://www.greenpeace.org.uk/latest-updates/?topic=air-pollution&entry_type) as stated in the ticket description.

Reply:
Okay, I'll look into it